scan/src/components/account/AccountIndexRedelegate.re

```diff
@@ -0,0 +1,129 @@
+module Styles = {
+  open Css;
+
+  let tableLowerContainer = style([padding(`px(10))]);
+
+  let hFlex = style([display(`flex)]);
+
+  let alignRight = style([display(`flex), justifyContent(`flexEnd)]);
+};
+
+[@react.component]
+let make = (~address) =>
+  {
+    let currentTime =
+      React.useContext(TimeContext.context)
+      |> MomentRe.Moment.defaultUtc
+      |> MomentRe.Moment.format("YYYY-MM-DDTHH:mm:ss.SSSSSS");
+
+    let (page, setPage) = React.useState(_ => 1);
+    let pageSize = 10;
+
+    let redelegateCountSub = RedelegateSub.getRedelegateCountByDelegator(address, currentTime);
+    let redelegateListSub =
+      RedelegateSub.getRedelegationByDelegator(address, currentTime, ~pageSize, ~page, ());
+
+    let%Sub redelegateCount = redelegateCountSub;
+    let%Sub redelegateList = redelegateListSub;
+
+    let pageCount = Page.getPageCount(redelegateCount, pageSize);
+    <div>
+      <VSpacing size=Spacing.md />
+      <div className=Styles.hFlex>
+        <HSpacing size=Spacing.lg />
+        <Text value={redelegateCount |> string_of_int} weight=Text.Semibold />
+        <HSpacing size=Spacing.xs />
+        <Text value="Redelegate Entries" />
+      </div>
+      <VSpacing size=Spacing.lg />
+      <>
+        <THead>
+          <Row>
+            <Col> <HSpacing size=Spacing.lg /> </Col>
+            <Col size=1.>
+              <Text
+                block=true
+                value="VALIDATOR"
+                size=Text.Sm
+                weight=Text.Bold
+                spacing={Text.Em(0.05)}
+                color=Colors.gray6
+              />
+            </Col>
```

Comment:
I think it's wrong. In redelegation, we'd like to display Source Validator, Destination Validator.